### PR TITLE
Using keyspace 'shoppingcartservice'

### PR DESCRIPTION
**Pull request by octonato**

### CODE REVIEW

File: docs-source/docs/modules/microservices-tutorial/examples/shopping-cart-service-scala/ddl-scripts/create_user_tables.cql [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 // tag::itemPopularityTables[]
-USE shoppingcart;
+USE shoppingcartservice;
 CREATE TABLE IF NOT EXISTS item_popularity (
```
Comment (patriknw) [inline docs-source/docs/modules/microservices-tutorial/examples/shopping-cart-service-scala/ddl-scripts/create_user_tables.cql:4]: there are at least 3 config properties also, `keyspace = shoppingcart`

Comment (octonato) [reply docs-source/docs/modules/microservices-tutorial/examples/shopping-cart-service-scala/ddl-scripts/create_user_tables.cql:4]: 🤦

Comment (octonato) [reply docs-source/docs/modules/microservices-tutorial/examples/shopping-cart-service-scala/ddl-scripts/create_user_tables.cql:4]: Now everybody knows I can't propertly search on text files.

File: docs-source/docs/modules/microservices-tutorial/examples/02-shopping-cart-service-scala/src/main/resources/persistence.conf [hunk 2]
```diff
@@ -27,7 +27,7 @@ akka {
   }
```
Comment (patriknw) [inline docs-source/docs/modules/microservices-tutorial/examples/02-shopping-cart-service-scala/src/main/resources/persistence.conf:27]: a few more things that are out of sync in this 02 config, but let me take care of that when splitting up the sources for real

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**